app/src/main/java/org/fossasia/pslab/adapters/AccelerometerAdapter.java

```diff
@@ -179,6 +147,47 @@ public void onAccuracyChanged(Sensor sensor, int accuracy) {
         }
     }
 
+    public AccelerometerAdapter(String[] dataset, Context context) {
+        this.dataset = dataset;
+        this.context = context;
+    }
+
+    @NonNull
+    @Override
+    public AccelerometerAdapter.ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
```

Comment:
These changes are due to reformatting.

Reply:
@harsh-2711 did reformatting the code shifted the whole `OnCreateViewHolder()` method below `OnBindViewHolder()` method?